src/components/StaffPayment/StaffPayment.vue

```diff
@@ -59,6 +60,8 @@
           <!-- No Fee radio button -->
           <v-radio id="no-fee-radio" label="No Fee" :value="StaffPaymentOptions.NO_FEE" />
 
+          <hr>
```

Comment:
What's the difference between a <br> and <hr>?

Reply:
It looks to me like they're different.
https://www.w3schools.com/howto/howto_css_style_hr.asp
https://vuetifyjs.com/en/components/dividers/

Therefore I think we should use `v-divider` here for consistency.